resources/darwin/bin/nerdctl

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
Maybe add

```
set -eu -o pipefail
```

so we catch any errors during future changes early?

Reply:
done

Reply:
This file should be stored elsewhere, like in `src/resources/darwin/nerdctl`, and copied in during `npm postinstall`. The problem is that `/resources/darwin` is in `.gitignore` and it's easy to overlook changes to the file, or worse, accidentally commit unintentional changes to the file because it didn't show up in a `git status`

Reply:
@epromislow That is not how `.gitignore` works; it only applies to files that are untracked. Once the file has been added to the repo, any changes to it will automatically be listed by `git status` and added by `git add -u` etc.

Reply:
The one disadvantage to putting .gitignored files into a repo is that they are by default also ignored by tools like `ag`, and `fd`.

Which makes me think that moving the file to a non-ignored location is still worthwhile, even though it makes no difference to git itself, just other tools that piggy-back on `.gitignore`, but don't double-check if the file is already tracked or not.